haskell-overlays/misc-deps.nix

```diff
@@ -38,4 +38,16 @@ in
   unliftio-core = self.callHackage "unliftio-core" "0.2.0.1" {};
   shelly = self.callHackage "shelly" "1.9.0" {};
   monad-logger = self.callHackage "monad-logger" "0.3.36" {};
+  nix-thunk = (import ../dep/nix-thunk {}).makeRunnableNixThunk (self.callCabal2nix "nix-thunk" (hackGet ../dep/nix-thunk) {});
+  cli-extras = self.callCabal2nix "cli-extras" (hackGet ../dep/cli-extras) {};
+  cli-git = haskellLib.overrideCabal (self.callCabal2nix "cli-git" (hackGet ../dep/cli-git) {}) {
+    librarySystemDepends = with pkgs; [
```

Comment:
will do. it's definitely necessary since we callCabal2nix it, but it might be avoidable if we make cli-git's default.nix the cabal2nix derivation, add the dependency there, and import the thunk rather than doing callCabal2nix on it.

Reply:
Opened https://github.com/obsidiansystems/cli-git/issues/3